kbase-extension/static/kbase/js/widgets/function_input/editors/kbaseModelEditor.js

```diff
@@ -0,0 +1,589 @@
+/**
+ *	kbaseModelEditor.js (kbaseModelEditor)
+ *
+ *	Given a pair of workspace and media object names or ids,
+ *	produce editable model tables.
+ *
+ * Authors:
+ * 	 nconrad@mcs.anl.gov
+ *
+ */
+
+(function( $, undefined ) {
```

Comment:
Should be wrapped in a define() statement, with the usual dependencies (kbwidget, kbaseAuthenticatedWidget, kbapi, kbmodeling, whatever else you've got)